examples/tree_1d_dgsem/elixir_mhd_shu_osher_shock_tube.jl

```diff
@@ -7,6 +7,55 @@ using Trixi
 gamma = 5/3
 equations = IdealGlmMhdEquations1D(gamma)
 
+"""
+    initial_condition_shu_osher_shock_tube(x, t, equations::IdealGlmMhdEquations1D)
+
+Extended version of the test of Shu and Osher for one dimensional ideal MHD equations.
+Taken from Section 4.1 of
+- Derigs et al. (2016)
+  A Novel High-Order, Entropy Stable, 3D AMR MHD Solver withGuaranteed Positive Pressure
+  [DOI: 10.1016/j.jcp.2016.04.048](https://doi.org/10.1016/j.jcp.2016.04.048)
+"""
+function initial_condition_shu_osher_shock_tube(x, t, equations::IdealGlmMhdEquations1D)
+  # domain must be set to [-5, 5], γ = 5/3, final time = 0.7
+  # initial shock location is taken to be at x = -4
+  x_0 = -4.0
+  rho = x[1] <= x_0 ? 3.5 : 1.0 + 0.2 * sin(5.0 * x[1])
+  v1 = x[1] <= x_0 ? 5.8846 : 0.0
+  v2 = x[1] <= x_0 ? 1.1198 : 0.0
+  v3 = 0.0
+  p = x[1] <= x_0 ? 42.0267 : 1.0
+  B1 = 1.0
+  B2 = x[1] <= x_0 ? 3.6359 : 1.0
```

Comment:
Same question on `<=` vs `<`

Reply:
@andrewwinters5000? You added this in 53fd46ad347eec5c14e119e4ec75b8b2671769d7

Reply:
This is the setup that Dominik and I used back in the day for the FLASH code. He found it in this PhD thesis https://uwspace.uwaterloo.ca/handle/10012/8597